src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -23,15 +21,10 @@
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_EMAIL + "johnd@example.com ";
```

Comment:
Do we need to add an IsDone field as well? Are we assuming that everyone we add has false for "IsDone"?

Reply:
I think we decided that everyone we added is considered to be not 'done'.

Reply:
ok sounds good!